2333/2/libs/top_bad_words_analyzer.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'handler'
+require 'terminal-table'
+require_relative './analyzer_printer.rb'
+require_relative './names_searcher.rb'
+
+# VersusBattle text analyzer
+class TopBadWordsAnalyzer
+  def initialize(rappers, take_value)
+    @rappers = rappers
+    @take_value = take_value.to_i
+  end
+
+  def top_bad_words
+    AnalyzerPrinter.print_top_bad_words(analyze_rappers)
+  end
+
+  private
+
+  def analyze_rappers
+    result = []
+    @rappers.first(@take_value).each do |rapper|
+      result << rapper.stats
+    end
+    result.sort_by! { |rapper| rapper[3] }.reverse!
```

Comment:
Compare `sort_by { |rapper| rapper[3] }` and `sort_by { |rapper| rapper.number_of_bad_words }`. What is more readable?

You don't need to have this weird `#stats` method of the rapper.